### ISSUE TITLE
Admob policy violation issue due to app open

### ISSUE THREAD

**Issue by merbin2012** [admob-plus/admob-plus#589]
Recently I received the following message from Admob

The solution
[App open policy violation issue fix solution](https://www.b4x.com/android/forum/threads/modified-ad-behavior-policy-violation-in-admob-with-app-open-ad.136835/)

"Modified ad behavior Must fix
Publishers are not permitted to alter the behavior or targeting of Google ads. This includes implementing the AdSense ad code in a "floating box script," or altering the ad-targeting using hidden keywords or IFRAMES."

After many searches, I found the exact problem. The app open ads should display in the following format 
Source [App open ad guidance](https://support.google.com/admob/answer/9341964?hl=en#zippy=%2Cdiscouraged-implementation-example%2Crecommended-implementation-example)
![app-open-ad](https://github.com/admob-plus/admob-plus/assets/3687184/0d54bfbf-13cd-40b8-8f2c-c7ae1a9f80ff)

But in our apps, we are displaying like the below one,
![erro openad 141221a1](https://github.com/admob-plus/admob-plus/assets/3687184/9736d45c-1337-46b2-aeb8-44a8e2bf43b1)

**Comment by merbin2012** [comment]
@ratson   I thought this problem occurred in the old version, but still found in the current version too. Sometimes it displays in the correct format.

Due to this problem, I have faced the following problem.

![screenshot-nimbusweb me-2023 06 27-09_47_04](https://github.com/admob-plus/admob-plus/assets/3687184/3f0504ed-f8c0-406c-8df9-2026d499e65f)

**Comment by ngdevr** [comment]
@merbin2012 , hi, is this still a problem? I have got policy violation mail from Google for the app using old version 1.28.0. I am wondering if I should upgrade to the latest. If the issue persists with the latest version as well then I would wait and simply disable appOpen ad for now.

**Comment by ngdevr** [comment]
@merbin2012 Thanks for the suggestion, I did update to the latest version and submitted the updated app for review to the policy review team. I will get back when I hear back from them. 

One issue I had to struggle with was orientation flag in AppOpen ads which I had set to admob.AppOpenAd.Orientation.LandscapeLeft. This seems to be automatic now. The example code still has reference to this though.

**Comment by merbin2012** [comment]
@ngdevr for me first they rejected it, but I approached them through the contact form, and after the manual verification, they approved the app.

If they reject this updated version too, you can contact them and tell "I fixed all the problems but was still rejected by the policy team. I am not sure, why this happened. So kindly send the screenshot and give me more information on this." 
Note: Check and change the grammar & corrections.

**Comment by ngdevr** [comment]
@merbin2012 Thanks, I just received an update from the policy team and they have again rejected the app. Which contact form are you referring to?

**Comment by merbin2012** [comment]
@ngdevr please check with google.
Check it
Did you send the app for review after live the app? 
Did you increase the app version and it's code?

**Comment by ngdevr** [comment]
@merbin2012 Yes, I sent it for review after the app went live. App version was definitely increased otherwise it would not have let me update the app in play store. To update further, after the second time rejection, I again sent a mail to policy team through "fix" option and asked them to provide me screenshot of what they say is a policy violation. Surprisingly, this time they simply approved saying that they did not find any policy violation.  This was my 3rd review request and when I asked to provide me with screenshot of the violation.
